### PR TITLE
Finished single goal progress endpoint

### PR DESCRIPTION
**Pull request by Destiny-01**

### CODE REVIEW

**Review by Adeniyii — CHANGES_REQUESTED** [review]
Hey man, please refactor to get those highlighted characters out of your code, thanks.

**Review by KelvinAkpobome — CHANGES_REQUESTED** [review]
Please check and correct the merge errors

File: controllers/goalController.js [hunk 1]
```diff
@@ -244,6 +238,17 @@ exports.createGoal = catchAsync(async (req, res, next) => {
       await insertOne('orgs', { orgId }, 'fictionalorganisationtokeeptrack');
     }
 
+<<<<<<< HEAD
```
Comment (Adeniyii) [inline controllers/goalController.js:241]: here

File: controllers/goalController.js [hunk 2]
```diff
@@ -244,6 +238,17 @@ exports.createGoal = catchAsync(async (req, res, next) => {
...
+    if (goals.data.status === 200) {
+      await createNotification(user_ids, orgId, roomId, title, 'createGoal');
+      logger.info(`Successfully created a new goal: ${goals.data.data}`);
+      res.status(200).json({ message: 'success', data });
+    }
+  } catch (error) {
+    return res.status(400).send({ message: 'Invalid request' });
+  }
+});
+=======
```
Comment (Adeniyii) [inline controllers/goalController.js:251]: here

File: controllers/targetController.js [hunk 3]
```diff
@@ -1,3 +1,14 @@
+<<<<<<< HEAD
```
Comment (Adeniyii) [inline controllers/targetController.js:1]: here

**Review by Adeniyii — COMMENTED** [review]
There are many more, please attend to them.

File: controllers/goalController.js (hunk 1 above)
Comment (Destiny-01) [reply controllers/goalController.js:241]: done

**Comment by KelvinAkpobome** [comment]
There are more. Please check and clean up

**Comment by Destiny-01** [comment]
Where @KelvinAkpobome ?

File: routes/goalRoutes.js [hunk 4]
```diff
@@ -16,31 +16,37 @@ const {
   disLikeGoal,
   getGoalDisLikes,
   checkUserDisLikes,
+<<<<<<< HEAD
```
Comment (KelvinAkpobome) [inline routes/goalRoutes.js:19]: Here

File: routes/goalRoutes.js [hunk 5]
```diff
@@ -1,6 +1,6 @@
 /* eslint-disable no-unused-vars */
 const { Router } = require('express');
-const {getChartInfo} = require('../controllers/chartController')
+const { getChartInfo } = require('../controllers/chartController');
 const {
```
Comment (KelvinAkpobome) [inline routes/goalRoutes.js:4]: Here too

**Review by KelvinAkpobome — COMMENTED** [review]
It's the goal routes that's left, that's all

File: controllers/goalController.js [hunk 6]
```diff
@@ -254,6 +259,7 @@ exports.createGoal = catchAsync(async (req, res, next) => {
       return res.status(400).send({ message: 'Invalid request' });
     }
   })
+>>>>>>> a2ddfd9e90f8792334f1e7fc5fb65f9ab28ec2c6
```
Comment (KelvinAkpobome) [inline controllers/goalController.js:263]: Here too

**Pull request closed without merging**